computer/login_server.py

```diff
@@ -1,15 +1,19 @@
 import paramiko, getpass
 from paramiko_server import ParamikoServer
+import time
+
 
 class LoginServer(ParamikoServer):
     def __init__(self, utup, cpus, roots, credentials):
         super(LoginServer, self).__init__(utup, cpus, roots, credentials)
-        self.password = getpass.getpass("Password for " + credentials['domain'] + ": ")
+        # self.password = getpass.getpass("Password for " + credentials['domain'] + ": ")
```

Comment:
The password can't be saved in this object in general.  If you want, this could be set in a conditional: `if not hasattr(self, 'password'):`